schedule_lessons/dashboard/static/dashboard/js/agenda.js

```diff
@@ -10,6 +10,10 @@ $(document).ready(function () {
     // $('#options_btn').toggleClass('addBtnAnim');
   });
 
+  $("#secondNavBtn").click(function(){
```

Comment:
nit/consistency note: I personally prefer single quotes in javascript code but depending on how we have done it everywhere else i am ok with double quotes as long as we are consistent :) 
also, just check this out for fun if you'd like: https://bytearcher.com/articles/single-or-double-quotes-strings-javascript/

Reply:
I'll make it consistent. Thanks for noticing